src/lurk/tag.rs

```diff
@@ -30,12 +30,12 @@ pub enum Tag {
     Key,
     Fun,
     Builtin,
+    Coroutine,
```

Comment:
Since hardcoded digests are changing, I took the chance to bring this tag closer to other symbol-like tags